miden-lib/src/transaction/procedures.rs

```diff
@@ -1,9 +1,22 @@
 use alloc::vec::Vec;
 
-use miden_objects::{digest, Digest, Felt, Hasher};
+use miden_objects::{utils::Deserializable, Digest, Felt, Hasher};
+use once_cell::sync::OnceCell;
 
 use super::TransactionKernel;
 
+// CONSTANTS
+// ================================================================================================
+
+/// Number of currently used kernel versions.
+const NUM_VERSIONS: usize = 1;
+
+/// Include file with kernel 0 procedure hashes generated in build.rs
+const PROCEDURES_RAW: [&[u8]; NUM_VERSIONS] = [include_bytes!("../../kernel_procs_v0.bin")];
```

Comment:
Related to one of the previous comments, it would be great to auto-generate a `.rs` file instead (e.g., something like `src/transaction/procedures/kernel_v0.rs` and import this constant from there.

I would also prefer to keep the it as `[Digest; 28]`, if possible.